config-files/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+### PR Checklist
+
+- [ ] JIRA Ticket(s) included in PR Title and are formatted correctly if applicable (JIRA-123)
+- [ ] PR Title is a short summary of your changes
+- [ ] PR Description includes details not included in the title summary
+- [ ] Assign applicable leads to your PR for review
+
+### Additional PR Detail
```

Comment:
How about just leaving line 9 out and saying ### Additional PR Details Below: or something similar on line 8?